### ISSUE TITLE
Duplicate point cloud for each camera

### ISSUE THREAD

**Issue by technicalartatk** [ByteDance-Seed/Depth-Anything-3#14]
Hello
Thanks for the great model

<img width="624" height="378" alt="Image" src="https://github.com/user-attachments/assets/fa983c42-b0cb-441f-8dc1-8727552ddac8" />
for the above input  im getting duplicate point cloud output for each camera , its not aligning point clounds together

<img width="1362" height="975" alt="Image" src="https://github.com/user-attachments/assets/f54cbd29-46bd-4020-807d-4d370de0226a" />

is there anything i have to do to fix this ,

**Comment by haotongl** [comment]
Hi there, thanks for your interest in our project! This issue may be somewhat outside our training distribution. I have two suggestions you can try to address your case:

1. Use the camera pose as a condition.
In the light stage, the camera poses should be fixed for each camera, so it’s reasonable to use them as an input condition. Please refer to the fourth demo in the Abilities section on our project page.

2. If matting is available, try setting the background to black.
This setup is more consistent with our training conditions for object scenes.

**Comment by AshadowZ** [comment]
Thanks to the authors for providing such a powerful baseline and very detailed documentation. I ran some tests on my own scenes and found that, just as mentioned in the paper, inputting the COLMAP intrinsics and extrinsics into the model greatly improves the geometric consistency of the reconstruction.

However, I found that with pure image input, the point cloud geometric consistency of Depth Anything 3 doesn't seem to be as good as Pi3's. I was wondering, in out-of-domain scenarios, did the authors compare the reconstruction results of DA3 and Pi3, and what were the results?

Here are my scenes: one is a Gundam model, and the other is a rather unconventional indoor scene with many artificial trees inside

<img width="1422" height="659" alt="Image" src="https://github.com/user-attachments/assets/10265c3b-f3dd-4635-a36f-831dcb51e3bd" />

<img width="1558" height="793" alt="Image" src="https://github.com/user-attachments/assets/860ae902-c724-4e15-8468-0afb5a601ed4" />

<img width="1101" height="908" alt="Image" src="https://github.com/user-attachments/assets/fc13eea8-66a3-4b12-ac8a-88bb97d48044" />

**Comment by technicalartatk** [comment]
Hello , thanks for the inputs. 
I tried making the background black(Transparent) , still the same result of duplicate point clouds , i think i should finetune the model to work with human dataset.

<img width="1295" height="750" alt="Image" src="https://github.com/user-attachments/assets/86981b08-9524-49bc-b28f-cc2fe3537f28" />

below are the inputs given:

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/81581be3-9f7f-4009-84d1-bcf0d76220c4" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/006d87d6-3754-4a75-9f0a-906c498958b1" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/bb54c47d-ec7c-47f5-b8e0-af2353dc5cc5" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/71036111-a42d-42de-85b9-f26001a88d59" />

**Comment by technicalartatk** [comment]
I tried VGGT , PI3  every model has the same issue of duplicating point cloud for each image input.,
Can someone please help on how to fix this issue.

**Comment by haotongl** [comment]
> Thanks to the authors for providing such a powerful baseline and very detailed documentation. I ran some tests on my own scenes and found that, just as mentioned in the paper, inputting the COLMAP intrinsics and extrinsics into the model greatly improves the geometric consistency of the reconstruction.
> 
> However, I found that with pure image input, the point cloud geometric consistency of Depth Anything 3 doesn't seem to be as good as Pi3's. I was wondering, in out-of-domain scenarios, did the authors compare the reconstruction results of DA3 and Pi3, and what were the results?
> 
> Here are my scenes: one is a Gundam model, and the other is a rather unconventional indoor scene with many artificial trees inside
> 
> <img alt="Image" width="1422" height="659" src="https://private-user-images.githubusercontent.com/82862115/514767643-10265c3b-f3dd-4635-a36f-831dcb51e3bd.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMyMjE2MjYsIm5iZiI6MTc2MzIyMTMyNiwicGF0aCI6Ii84Mjg2MjExNS81MTQ3Njc2NDMtMTAyNjVjM2ItZjNkZC00NjM1LWEzNmYtODMxZGNiNTFlM2JkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTE1VDE1NDIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwMTMxMjAyODYxNDkzMTIxNzczODIzMWFjMzk2Yjk5MzgxZDg2OWZhMzdlZjFhNTZlMDkzYWE2M2I4N2YyYzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.C4PiBXAxgcWRom3sDGNgwyWBLc-_7_90-Q_tb4t5t14"> <img alt="Image" width="1558" height="793" src="https://private-user-images.githubusercontent.com/82862115/514767567-860ae902-c724-4e15-8468-0afb5a601ed4.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMyMjE2MjYsIm5iZiI6MTc2MzIyMTMyNiwicGF0aCI6Ii84Mjg2MjExNS81MTQ3Njc1NjctODYwYWU5MDItYzcyNC00ZTE1LTg0NjgtMGFmYjVhNjAxZWQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTE1VDE1NDIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQzMzU3NzI3MDZkZTkxZWUwMjc3OWJkMjE1NTMwMTY5YjFmNzZjMDhmMjkwNjIxODk4MTcyZjQyODNiY2UwYjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xTibM3x6G2DeJ5FfxKV9qTwls65wTMnw-91hSnBeedw"> <img alt="Image" width="1101" height="908" src="https://private-user-images.githubusercontent.com/82862115/514767830-fc13eea8-66a3-4b12-ac8a-88bb97d48044.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMyMjE2MjYsIm5iZiI6MTc2MzIyMTMyNiwicGF0aCI6Ii84Mjg2MjExNS81MTQ3Njc4MzAtZmMxM2VlYTgtNjZhMy00YjEyLWFjOGEtODhiYjk3ZDQ4MDQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTE1VDE1NDIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI0ODc5YTA4ZTYxZmMwNTgwODFiY2I3MjZhNzRlNGJiZDEzNjdkNGRjMDkyODcxNTQzMzA4Y2NlMGZmZWZjMzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.S9acSBaRoKDZwpHqmus9ukr0OCLtxj1_cH9MWYeTNnM">

Hi thanks for your attention. I observe that you take portrait (vertical) images as input. One possible reason is that our model was mainly trained on landscape images, which leads to a performance drop on portrait (vertical) images.

**Comment by haotongl** [comment]
> finetune

Yeah, I also think so. You may finetune our model on your human dataset.

**Comment by surpoloyang** [comment]
> I tried VGGT , PI3 every model has the same issue of duplicating point cloud for each image input., Can someone please help on how to fix this issue.

I also have the same question.

**Comment by jianglh-WHU** [comment]
"One possible reason is that our model was mainly trained on landscape images"
But in DA3 paper, the training image resolutions are randomly sampled from 504 × 504, 504 × 378, 504 × 336, 504 × 280, 336 × 504, 896 × 504, 756 × 504, 672 × 504. I think there still exists vertical images during training.

**Comment by kaikai23** [comment]
> > finetune
> 
> Yeah, I also think so. You may finetune our model on your human dataset.

Hi, may you have plans to release the training/ finetune code?
